### PR TITLE
Remove "@skip_screen_checks" to run the telenium test case

### PR DESCRIPTION
**Pull request by shekhar-cis**
Remove "@skip_screen_checks" to run the telenium test case for Generating a new identity.
**Expected fail because we have not merged the updated `mpybit.py`. and #1950 myaddress screen**

### CODE REVIEW

**Review by PeterSurda — COMMENTED** [review]
We'll see the results after merging the pending prerequisites.

**Comment by PeterSurda** [comment]
this needs rebasing

**Review by PeterSurda — APPROVED** [review]

File: src/bitmessagekivy/tests/test_create_random_address.py [hunk 1]
```diff
@@ -60,22 +65,21 @@ def test_generate_random_address_label(self):
...
-        # Checking the dropdown option is exist
-        self.assertExists('//MySpinnerOption[0]', timeout=5)
-        is_open = self.cli.getattr('//NavigationItem[0]/CustomSpinner[@is_open]', 'is_open')
-        # Check the state of dropdown.
-        self.assertEqual(is_open, True)
-        # Selection of an address to set as a default address.
-        self.cli.wait_click('//MySpinnerOption[0]', timeout=5)
-        # Checking current screen
-        self.assertExists("//ScreenManager[@current=\"inbox\"]", timeout=5)
+    # @ordered
```
Comment (PeterSurda) [inline src/bitmessagekivy/tests/test_create_random_address.py:68]: you can just ignore this chunk

**Review by PeterSurda — CHANGES_REQUESTED** [review]
instead of commenting out the raise Timeout, change the `break` to `return`

File: src/bitmessagekivy/tests/test_create_random_address.py [hunk 2]
```diff
@@ -24,18 +25,22 @@ def test_landing_screen(self):
...
-        start = time()
-        deadline = start + 2
-        while time() < deadline:
-            try:
-                # Clicking on Proceed Next Button to redirect to "random" screen
-                self.cli.wait_click('//Screen[0]//MDFillRoundFlatIconButton[@text=\"Proceed Next\"]', timeout=5)
-            except TeleniumHttpException:
-                # Checking Current Screen(Random Screen) after Clicking on "Proceed Next" Button
-                self.assertExists("//ScreenManager[@current=\"random\"]", timeout=5)
+        self.assertExists("//Screen[0]//MDFillRoundFlatIconButton[@text=\"Proceed Next\"]", timeout=5)
```
Comment (PeterSurda) [inline src/bitmessagekivy/tests/test_create_random_address.py:28]: just leave this as it is after fixing the timeout exception

File: src/bitmessagekivy/tests/telenium_process.py [hunk 3]
```diff
@@ -81,7 +81,7 @@ def assert_wait_no_except(self, selector, timeout=-1, value='inbox'):
                 # Finally Sleep is used to make the menu button funcationlly available for the click process.
                 # (because Transition is little bit slow)
                 sleep(0.2)
-        raise AssertionError("Timeout")
+            raise AssertionError("Timeout")
```
Comment (PeterSurda) [inline src/bitmessagekivy/tests/telenium_process.py:84]: this still isn't fixed correctly

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/bitmessagekivy/tests/telenium_process.py (hunk 3 above)
Comment (shekhar-cis) [reply src/bitmessagekivy/tests/telenium_process.py]: Now is it correct?

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**